steps/issue-create/spec.schema.json

```diff
@@ -0,0 +1,103 @@
+
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "connection": {
+      "type": "object",
+      "x-relay-connection-type": "jira",
```

Comment:
Should be `x-relay-connectionType` (JSON Schema uses camelCase, otherwise I wouldn't have made that choice).

Reply:
Great thanks! Fixing.